static/app/components/dropdownMenuControlV2.tsx

```diff
@@ -0,0 +1,193 @@
+import {
+  ElementType,
+  HTMLAttributes,
+  MouseEvent,
+  ReactNode,
+  RefObject,
+  useRef,
+} from 'react';
+import styled from '@emotion/styled';
+import {useButton} from '@react-aria/button';
+import {AriaMenuOptions, useMenuTrigger} from '@react-aria/menu';
+import {AriaPositionProps, OverlayProps} from '@react-aria/overlays';
+import {Item, Section} from '@react-stately/collections';
+import {useMenuTriggerState} from '@react-stately/menu';
```

Comment:
Did we already add `react-stately` for something else?

Reply:
Ahh this comes with `react-aria` huh? We should specify it as a direct dependency in our package.json in that case since we're using it.

Reply:
I don't think so (?) `react-stately` is a dependency for `react-aria`, so both are installed when add `react-aria`

Reply:
@vuluongj20 I agree w/ @evanpurkhiser, we should add all libraries that we directly import in `package.json`. e.g. yarn v2 will error on this.